Area51/Sources/ContributionsController.swift

```diff
@@ -0,0 +1,48 @@
+import UIKit
+
+class ContributionController: UITableViewController {
+
+    var listOfContributors = [Contributor]()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(ContributorCell.nib,
+                           forCellReuseIdentifier: ContributorCell.reuseIdentifier)
+        self.title = "Contributors"
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        DispatchQueue.main.async {
```

Comment:
since `getContributors` uses URLSession dataTask, we do not need to dispatch to main here but in the completion handler:

```
    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        Contributor.getContributors { (res) in
            DispatchQueue.main.async {
                switch res {
                case .success(let contributors):
                    contributors.forEach({ (contributor) in
                        self.listOfContributors.append(contributor)
                    })
                    self.tableView.reloadData()
                case .failure(let err):
                    print(err)
                }
            }
        }
    }
```

Reply:
Adjusted! Thank you for helping me realize that: 

> The URL session API itself is fully thread-safe. You can freely create sessions and tasks in any thread context.